javaparser-core/src/main/java/com/github/javaparser/ast/body/MethodDeclaration.java

```diff
@@ -158,11 +158,6 @@ public MethodDeclaration setName(final SimpleName name) {
     }
 
     @Override
-    public MethodDeclaration setParameters(final NodeList<Parameter> parameters) {
```

Comment:
This code was added before to handle receiver annotations. Then it was moved to the super class setParameters method.